### PR TITLE
feat: add live query plugin

### PR DESCRIPTION
**Pull request by n1ru4l**

### CODE REVIEW

File: packages/plugins/live-query/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      // @ts-expect-error: execute typings do not include AsyncIterable return right now
+      setExecuteFn(opts.liveQueryStore.makeExecute(executeFn));
+    },
+    onValidate: ({ addValidationRule }) => {
+      addValidationRule(NoLiveMixedWithDeferStreamRule);
+    },
+    onSchemaChange: ({ schema, replaceSchema }) => {
+      let patchedSchema = preparedSchemas.get(schema);
+      if (!patchedSchema) {
+        patchedSchema = addTypes(schema, [GraphQLLiveDirective]);
```
Comment (n1ru4l) [inline packages/plugins/live-query/src/index.ts]: Not sure whether adding the directive should be scope of the envelop tbh

Comment (dotansimha) [reply packages/plugins/live-query/src/index.ts]: I think we can expose it as string from the plugin package, so it will be easier to add it to the schema

Comment (dotansimha) [reply packages/plugins/live-query/src/index.ts]: And we can check in onSchemaChange if the schema has it or not

**Review by dotansimha — APPROVED** [review]

File: packages/plugins/live-query/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+```bash
+yarn add @envelop/live-query @n1ru4l/in-memory-live-query-store
+```
+
+## Usage Example
+
+```ts
+import { envelop, useSchema, useExtendContext } from '@envelop/core';
+import { createApplication } from 'graphql-modules';
```
Comment (dotansimha) [inline packages/plugins/live-query/README.md]: unsued?

File: packages/plugins/live-query/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+```tsx
+import { GraphQLSchema } from 'graphql';
+import { GraphQLLiveDirective } from '@envelop/live-query';
+
+const schema = new GraphQLSchema({
+  directives: [...specifiedDirectives, GraphQLLiveDirective],
+});
+```
+
+## Further information
```
Comment (dotansimha) [inline packages/plugins/live-query/README.md:84]: Maybe add a short section about transports, and some links to examples ?

Comment (n1ru4l) [reply packages/plugins/live-query/README.md:84]: done :)

**Pull request merged**